gaia/core/task.py

```diff
@@ -166,16 +212,16 @@ class Source(Task):
 
             """Generated source task."""
 
-            output_ports = {'': SourceOutput}
+            output_ports = {'0': SourceOutput}
 
-            def __init__(self, *arg, **kw):
-                """Initialize a source task."""
-                super(Source, self).__init__(*arg, **kw)
-                self._output_data[''] = data
+            def get_input_data(self, name='0'):
```

Comment:
should this be private (with underscore) since we don't have set_input_data. Looks like it will be used internally by the program and not by the user?